Src/Support/Google.Apis.Auth/OAuth2/ComputeCredential.cs

```diff
@@ -237,10 +237,15 @@ private static async Task<bool> IsRunningOnComputeEngineNoCache()
                     }
                     catch (Exception e) when (e is HttpRequestException || e is WebException || e is OperationCanceledException)
                     {
-                        // No-op; we'll retry.
+                        // We'll retry, but let's log the exception.
                         // We may eventually want to handle the different exception types in different ways,
                         // e.g. returning false rather than retrying for some exception types. However,
                         // for now it's safe just to retry.
+                        Logger.Debug("An exception ocurred while attempting to reach Google's metadata service on attempt {0}." +
```

Comment:
Please add a space at the end of each line. I'd be tempted to put the `{2}` on the same source line as "The exception was:" as well.

Reply:
Both done! Pushing and merging on green. Thanks!